src/components/UI/Overlay/OverlayContent/DocumentTransferMessage.tsx

```diff
@@ -15,6 +15,7 @@ export enum MessageTitle {
   SURRENDER_DOCUMENT_SUCCESS = "Surrender Document Success",
   ACCEPT_SURRENDER_DOCUMENT = "Surrender Accepted",
   REJECT_SURRENDER_DOCUMENT = "Surrender Rejected",
+  CONFIRM_REJECT_SURRENDER_DOCUMENT = "Confirm Reject Surrender",
```

Comment:
```suggestion
  CONFIRM_REJECT_SURRENDER_DOCUMENT = "Confirm Document Return",
```

Just a suggestion on the wording... not sure if its better though, what do you think?

Reply:
?